.travis.yml

```diff
@@ -1,26 +1,34 @@
-dist: trusty
 language: node_js
```

Comment:
I don't really think, it's slow now, but I can't tell, if it would make a difference.
I tried to keep everything as brief as possible. I will try this once.

Reply:
Hmmm, I'm not sure, if and how adding 
```
sudo: false 
dist: trusty
```
would affect `osx` builds: https://docs.travis-ci.com/user/reference/overview/#virtualisation-environment-vs-operating-system
Would they be still be possible? Do I have to make different settings for them?

I'm currently preparing a different PR for Windows builds, and I'm actually trying if I could even build Windows binaries with Travis instead of AppVeyor. It's the recommendation by `electron-builder` to use Travis, if it's possible to use Wine.

So I would prefer to leave the current PR as it is. Maybe the whole build matrix will become more complex anyway with Windows support, because it will run in Docker and then does the sudo-enabled environment anyway.